redact/redact_requests.py

```diff
@@ -58,23 +58,18 @@ def __init__(
         api_key: Optional[str] = None,
         httpx_client: Optional[httpx.Client] = None,
     ):
-
         self.redact_url = normalize_url(redact_url)
         self.api_key = api_key
         self.subscription_id = subscription_id
         self._headers = {"Accept": "*/*"}
-        self.retry_total_time_limit: float = 600  # 10 minutes in seconds
```

Comment:
Just for my understanding: Why remove comment?

Reply:
Sorry I think it was an accident when merging. I will add it back in.

Reply:
done